packages/widgets/src/menu.ts

```diff
@@ -29,6 +29,7 @@ import {
   VirtualDOM,
   VirtualElement
 } from '@lumino/virtualdom';
+import { MenuBar } from './menubar';
```

Comment:
```suggestion
import type { MenuBar } from './menubar';
```

This lets the code reader know that we're not actually creating `MenuBar` instances in this file, just using it for type definitions.